seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/AuthController.java

```diff
@@ -41,8 +41,8 @@ public class AuthController {
     @GetMapping("/userRole")
     @ApiOperation(value = "check relation between user and role", httpMethod = "GET")
     @ApiImplicitParams({
-            @ApiImplicitParam(name = "username", value = "user name", dataType = "String"),
-            @ApiImplicitParam(name = "roleName", value = "role name", dataType = "String"),
+        @ApiImplicitParam(name = "username", value = "user name", dataType = "String"),
```

Comment:
revert?

Reply:
This is to fix the code format problem reported by `checkstyle`.